packages/core/src/components/Footer/About.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Create this component directly under `src`... This is already a **components** library.